src/locales/nl_NL/nl_NL.json

```diff
@@ -0,0 +1,447 @@
+            "full_name": "Full name",
+            "issue_date": "Issue date or summary period",
+            "label": "Make sure it clearly shows:",
+            "logo": "Logo"
+        },
+        "subtitle_bank_statement": "Must be issued in the <strong>last 3 months<\/strong>",
+        "subtitle_benefits_letter": "Must be issued in the <strong>last 12 months<\/strong>",
+        "subtitle_bill": "Must be issued in the <strong>last 3 months<\/strong>",
+        "subtitle_government_letter": "Must be issued in the <strong>last 12 months<\/strong>",
+        "subtitle_tax_letter": "Must be issued in the <strong>last 12 months<\/strong>"
+    },
+    "poa_intro": {
+        "button_primary": "Start verification",
+        "list_matches_signup": "<strong>Matches<\/strong> the address you used on signup",
+        "list_most_recent": "Is your most <strong>recent<\/strong> document",
+        "list_shows_address": "Shows your <strong>current<\/strong> address",
+        "subtitle": "You’ll need a document that:",
+        "title": "Let’s verify your %{country} address"
+    },
+    "selfie_capture": {
+        "alert": {
+            "camera_inactive": {
+                "detail": "Controleer dat deze verbinding heeft en werkt. U kunt <fallback>het verificatieproces ook op uw mobiele telefoon voortzetten<\/fallback>",
+                "detail_no_fallback": "Zorg dat de camera op uw apparaat werkt",
+                "title": "Doet uw camera het niet?"
+            },
+            "camera_not_working": {
+                "detail": "Hij is mogelijk losgekoppeld. <fallback>Probeer het met uw telefoon<\/fallback>.",
+                "detail_no_fallback": "Zorg dat de camera op uw apparaat werkt",
+                "title": "Camera doet het niet"
+            },
+            "timeout": {
+                "detail": "Vergeet niet op de knop te drukken als u klaar bent. <fallback>Herhaal de stappen voor het opnemen van een filmpje<\/fallback>",
+                "title": "Het lijkt erop dat het te lang duurde"
+            }
+        },
+        "body": "Houd uw gezicht binnen de ovaal",
+        "button_accessibility": "Neem een foto",
+        "button_primary": "Start een gezichtsscan",
+        "frame_accessibility": "Waargave vanaf camera",
+        "title": "Houd uw gezicht binnen de ovaal"
+    },
+    "selfie_confirmation": {
+        "image_accessibility": "Foto van uw gezicht",
+        "subtitle": "Zorg dat uw volledige gezicht zichtbaar is",
+        "title": "Controleer uw selfie"
+    },
+    "selfie_intro": {
+        "button_primary": "Doorgaan",
+        "list_accessibility": "Tips voor een goede selfie",
+        "list_item_face_forward": "Kijk recht vooruit en zorg dat uw ogen goed te zien zijn",
+        "list_item_no_glasses": "Zet uw bril af indien nodig",
+        "subtitle": "We vergelijken deze met uw document",
+        "title": "Neem een selfie"
+    },
+    "sms_sent": {
+        "info": "Tips",
```

Comment:
Is this dutch?

Reply:
According to Google Translate "Tips" is Dutch for "Tips". 😅  
I was talking to Rae yesterday about whether we might want someone who can actually read Dutch to also go through the translations since between us in the team I don't think any of us do.

Reply:
Good point, definitely not me. Meanwhile, I will approve it as it looks good to me.

Reply:
I just asked at my company if we have a Dutch copywriter who can help in reviewing this quickly for you to speed up the process of merging this PR bcz we definitely need it soon.

Reply:
@abbathaw If you are able to help with reviewing it also that would be very helpful & much appreciated. 😃